Readme.markdown

```diff
@@ -1,6 +1,8 @@
 # DVR
 
-[![Version](https://img.shields.io/github/release/venmo/DVR.svg)](https://github.com/venmo/DVR/releases) [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
+[![Version](https://img.shields.io/github/release/venmo/DVR.svg)](https://github.com/venmo/DVR/releases)
+[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
+[![Swift Package Manager](https://rawgit.com/jlyonsmith/artwork/master/SwiftPackageManager/swiftpackagemanager-compatible.svg)](https://swift.org/package-manager/)
```

Comment:
This is probably the most minor of minor nitpicks ever, but the design of those two badges doesn't match up. How about using shields.io's custom badges for spm compatibility? 😊 E.g. https://img.shields.io/badge/SPM-compatible-brightgreen.svg

Reply:
Nitpick all you want @kiliankoe  ;-)

Pushed with the badge you suggested. Thanks for the quick review!